### DISCUSSION TITLE
Can automa support switch iframe and window?

### DISCUSSION THREAD

**Discussion by moveurbody** [AutomaApp/automa#38] [Ideas]
I try to locate a button element in an iframe, but I can't use the element selector to locate the button element.
In selenium, it provides switch iframe and detects window. I think if you can support these functions will be awesome.

**Comment by Kholid060** [comment]
I'll try to add this feature in the future update

**Reply by ABrandt91** [reply]
Yes please! I have the same problem

**Comment by Kholid060** [comment]
The extension is now supported to switch between the main window and the iframe by using the "Switch frame" block, it's available on v0.5.1

**Reply by gaozjha** [reply]
Switch frame block can be used only twice，the wanted web has many layers of frame。｜》 can be used only once

**Comment by luo57wuqi** [comment]
yestoday,i want to loop element of leve7 iframe,by many switch iframe block,failure.
so ,i  let claude help to imporve it ,now it works
